test/deprecated-keyword/handleConfig.set.good

```diff
@@ -0,0 +1,4 @@
+handleConfig.chpl:5: In function 'main':
+handleConfig.chpl:6: warning: don't use x, use y
+true
+<command-line arg>:1: warning: don't use x, use y
```

Comment:
Is this the actual order that things print out, or is this reflecting some artifact of our test system w.r.t. stderr and stdout?  (I would've expected the command line warning to occur before `true` printed out).

Reply:
It's an artifact of our test system, running it manually printed the warning first